model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -47,17 +85,43 @@ def __init__(
         """
 
         self.validate(dist)
-
         self.dist = dist
         self.varname = varname
 
         return None
 
     @staticmethod
     def validate(distr: dist.Distribution) -> None:
+        """
+        Validates distribution is Numpyro distribution
+
+        Parameter
+        ---------
+        distr : dist.Distribution
+            A ingested distribution (e.g., prior IHR distribution)
+
+        Raises
+        ------
+        AssertionError
+            If the inputted distribution is not a Numpyro distribution.
+        """
         assert isinstance(distr, dist.Distribution)
 
     def sample(self, **kwargs) -> InfectHospRateSample:
+        """
+        Produces a sample of the IHR
+
+        Parameter
```

Comment:
Make sure is `Parameters` 😄 

```suggestion
        Parameters
```

Reply:
Solved in 898ac6c